src/components/Combobox/components/DropdownItems/DropdownItem.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+export const DropdownItem = (props) => {
```

Comment:
let's call it `DropdownItemCore` analogous to `ComboboxCore`.
what do you think?

Reply:
Good point, will do